apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/admin/api/client/BftScanConnection.scala

```diff
@@ -1015,16 +1026,100 @@ object BftScanConnection {
       }
     }
 
-    override def scanConnections: ScanConnections = {
+    protected def currentConnectionsState: ScanConnections =
       currentScanConnectionsRef.get().scanConnections
-    }
+
+    def scanConnections: ScanConnections
 
     override protected def closeAsync(): Seq[AsyncOrSyncCloseable] =
       initialScanConnections.zipWithIndex.map { case (connection, i) =>
         SyncCloseable(s"scan_connection_$i", connection.close())
       }
   }
 
+  class BftAny(
+      override val initialScanConnections: Seq[SingleScanConnection],
+      override val initialFailedConnections: Map[Uri, Throwable],
+      override val connectionBuilder: Uri => Future[SingleScanConnection],
+      override val getScans: BftScanConnection => Future[Seq[DsoScan]],
+      override val scansRefreshInterval: NonNegativeFiniteDuration,
+      override val retryProvider: RetryProvider,
+      override val loggerFactory: NamedLoggerFactory,
+  )(implicit override val ec: ExecutionContext)
+      extends Bft {
+
+    override protected def filterScans(allScans: Seq[DsoScan]): Seq[DsoScan] = allScans
+
+    override protected def validateState(state: BftState): Unit = {
+      val defaultCallConfig = BftCallConfig.default(state.scanConnections)
+
+      if (!defaultCallConfig.enoughAvailableScans) {
+        throw io.grpc.Status.FAILED_PRECONDITION
+          .withDescription(
+            s"There are not enough Scans to satisfy f=${state.scanConnections.f}. Will be retried. State: $state.scanConnections"
+          )
+          .asRuntimeException()
+      }
+    }
+
+    override def scanConnections: ScanConnections = {
+      val connections = currentConnectionsState
+      connections.copy(totalInSubset = None, threshold = None)
+    }
+  }
+
+  class BftCustom(
+      val trustedSvs: NonEmptyList[String],
+      val threshold: Option[Int],
+      override val initialScanConnections: Seq[SingleScanConnection],
+      override val initialFailedConnections: Map[Uri, Throwable],
+      override val connectionBuilder: Uri => Future[SingleScanConnection],
+      override val getScans: BftScanConnection => Future[Seq[DsoScan]],
+      override val scansRefreshInterval: NonNegativeFiniteDuration,
+      override val retryProvider: RetryProvider,
+      override val loggerFactory: NamedLoggerFactory,
+  )(implicit override val ec: ExecutionContext, tc: TraceContext)
+      extends Bft {
+
+    private val trustedSvsSet = trustedSvs.toList.toSet
+
+    override protected def filterScans(allScans: Seq[DsoScan]): Seq[DsoScan] = {
+      val targetScans = allScans.filter(scan => trustedSvsSet.contains(scan.svName))
+
+      logger.info(s"Discovered the following trusted scans from the network: ${targetScans
```

Comment:
I would avoid this as a info log if it's logged during every refresh, we could lower it to debug. 
We should log at info level only during the init the SVs that we're going to use.